modules/conversantBidAdapter.js

```diff
@@ -248,6 +248,46 @@ export const spec = {
       'secure': 'number',
       'mobile': 'number'
     }, params);
+  },
+
+  /**
+   * Register User Sync.
+   */
+  getUserSyncs: function(syncOptions, responses, gdprConsent, uspConsent) {
+    let params = {}
+    const syncs = [];
+
+    // Attaching GDPR Consent Params in UserSync url
+    if (gdprConsent) {
+      params.gdpr = (gdprConsent.gdprApplies) ? 1 : 0;
+      params.gdpr_consent = encodeURIComponent(gdprConsent.consentString || '');
+    }
+
+    // CCPA
+    if (uspConsent) {
+      params.us_privacy = encodeURIComponent(uspConsent);
+    }
+
+    if (responses && responses.ext) {
+      const pixels = [{urls: responses.ext.fsyncs, type: 'iframe'}, {urls: responses.ext.psyncs, type: 'image'}]
+        .filter((entry) => {
+          return entry.urls &&
+            ((entry.type === 'iframe' && syncOptions.iframeEnabled) ||
+            (entry.type === 'image' && syncOptions.pixelEnabled));
+        })
+        .flatMap((entry) => {
+          return entry.urls.flatMap((endpoint) => {
```

Comment:
This seems to be one too many flatMaps. Tests will pass when removing line 279 and 286. Also updating from endpoint to entry.urls on line 280.

Reply:
@johnwier fyi